packages/esm-patient-banner-app/src/banner-tags/deceased-tag.component.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import styles from './active-visit-tag.scss';
+import { useTranslation } from 'react-i18next';
+import { Tag, TooltipDefinition } from 'carbon-components-react';
+
+interface DecasedBannerTagProps {
+  patient: fhir.Patient;
+}
+const DeceasedBannerTag: React.FC<DecasedBannerTagProps> = ({ patient }) => {
+  const { t } = useTranslation();
+  return (
+    patient.deceasedBoolean && (
+      <TooltipDefinition
+        align="end"
+        tooltipText={
+          <div className={styles.tooltipPadding}>
+            <h6 style={{ marginBottom: '0.5rem' }}>deceased</h6>
```

Comment:
Did you want to translate this?

Reply:
I have translated it with the latest commit. @dkayiwa